user-access-internal-model/src/test/java/no/unit/nva/useraccessmanagement/internals/UserScanResultTest.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.useraccessmanagement.internals;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomBoolean;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import java.util.List;
+import java.util.Map;
+import no.unit.nva.useraccessmanagement.model.UserDto;
+import org.junit.jupiter.api.Test;
+
+class UserScanResultTest {
+
+    @Test
+    void shouldReturnObjectContainingInputArguments() {
```

Comment:
The class belongs to a shared module and should either be annotated with `@JacocoGenerated` or write a test